internal/flags/gitlab_visibility.go

```diff
@@ -51,16 +51,23 @@ func (d *GitLabVisibility) Set(str string) error {
 	}
 	var visibility = gitprovider.RepositoryVisibility(str)
 	if ValidateRepositoryVisibility(visibility) != nil {
-		return fmt.Errorf("unsupported visibility '%s'", str)
+		return fmt.Errorf("unsupported visibility '%s', must be one of: %s", str, d.Type())
 	}
 	*d = GitLabVisibility(visibility)
 	return nil
 }
 
 func (d *GitLabVisibility) Type() string {
-	return "gitLabVisibility"
+	visibilities := make([]string, 0, len(supportedGitLabVisibilities))
+	for visibility := range supportedGitLabVisibilities {
+		visibilities = append(visibilities, string(visibility))
+	}
+	return strings.Join(visibilities, "|")
 }
 
 func (d *GitLabVisibility) Description() string {
-	return fmt.Sprintf("specifies the visibility of the repository. Valid values are public, private, internal")
+	return fmt.Sprintf(
+		"specifies the visibility of the repository, available options are: (%s)",
```

Comment:
It seems that the `available options are: (...|...)` could potentially be removed, as this information is already specified in the type definition, making it somewhat redundant.